lib/github_advisory_sync.rb

```diff
@@ -0,0 +1,281 @@
+require "faraday"
+require "json"
+require "yaml"
+require "open-uri"
+
+module GitHub
+  class GitHubAdvisorySync
+
+    # Sync makes sure there are rubysec advisories for all GitHub advisories
+    # It writes a set of yaml files, one for each GitHub Advisory that
+    # is not already present in this repo
+    #
+    # The min_year argument specifies the earliest year CVE to sync
+    # There are many old CVEs in the GitHub advisory dataset that are not in here
+    # It is more important to sync the newer ones, so this allows the user to
+    # control how old of CVEs the sync should pull over
+    def self.sync(min_year: 2018)
```

Comment:
How many does the script output for 2018 onward?

Reply:
For 2018 onward, it produces this output:
```
Sync completed
Wrote these files:
---
- gems/rubygems-update/CVE-2019-8325.yml
- gems/rubygems-update/CVE-2019-8320.yml
- gems/rubygems-update/CVE-2019-8324.yml
- gems/rubygems-update/CVE-2019-8321.yml
- gems/rubygems-update/CVE-2019-8322.yml
- gems/rubygems-update/CVE-2019-8323.yml
- gems/ruby-openid/CVE-2019-11027.yml
- gems/chloride/CVE-2018-6517.yml
- gems/doorkeeper-openid_connect/CVE-2019-9837.yml
- gems/fat_free_crm/CVE-2018-1000842.yml
- gems/smart_proxy_dynflow/CVE-2018-14643.yml
- gems/jekyll/CVE-2018-17567.yml
- gems/bootstrap/CVE-2018-14042.yml
```

The six for rubygems are actually already in this repo over at https://github.com/rubysec/ruby-advisory-db/tree/master/libraries/rubygems

so it is syncing over 13 CVEs, but only 7 are legit additions I think.

If we look at at all time, the stats work out this:
```
  8 2011
  11 2012
  15 2013
  11 2014
   1 2015
   1 2017
   5 2018
   8 2019
```

All that is simply CVEs, which is all that is currently published by GitHub.  The real gain is the future...